lib/utils.js

```diff
@@ -68,6 +69,9 @@ module.exports.parseDSN = function parseDSN(dsn) {
           host: parsed.host.split(':')[0]
         };
 
+    var timeout = qs.parse(parsed.query).timeout;
```

Comment:
This ties into the "should we configure raven clients via DSN querystring params or via config fields and how should we standardize that stuff" discussion; I originally followed the lead of [raven-java](https://docs.sentry.io/clients/java/config/) and [raven-python](https://github.com/getsentry/raven-python/blob/master/docs/transports.rst) which both do it via DSN, but we don't really have any precedent of doing DSN-based config in raven-node. Open for discussion.